### PR TITLE
Poke

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Resurrected code from my second month here. Let's see how it's fared in the cellar.

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/OrganizationMembershipPoke.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class OrganizationMembershipPoke(
+    id: Option[Id[OrganizationMembershipPoke]] = None,
+    createdAt: DateTime = currentDateTime,
+    updatedAt: DateTime = currentDateTime,
+    state: State[OrganizationMembershipPoke] = OrganizationMembershipPokeStates.ACTIVE,
+    organizationId: Id[Organization],
+    userId: Id[User]) extends ModelWithState[OrganizationMembershipPoke] {
```
Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/model/OrganizationMembershipPoke.scala:16]: Maybe an `Either[Id[User], EmailAddress]`? Do we want to let non-users request an invite?

File: kifi-backend/modules/common/conf/evolutions/shoebox/409.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	state varchar(20) NOT NULL,
+	organization_id bigint(20) NOT NULL,
+	user_id bigint(20) NOT NULL,
+
+	PRIMARY KEY(id),
+	CONSTRAINT `organization_membership_poke_f_organization` FOREIGN KEY (`organization_id`) REFERENCES organization(`id`),
+	CONSTRAINT `organization_membership_poke_f_user` FOREIGN KEY (`user_id`) REFERENCES user(`id`)
+);
+
+insert into evolutions(name, description) values('357.sql', 'added organization_membership_poke');
```
Comment (ryanpbrewster) [inline kifi-backend/modules/common/conf/evolutions/shoebox/409.sql:18]: oops

**Comment by camhashemi** [comment]
:+1: lgtm, nice going

**Comment by ryanpbrewster** [comment]
This is going back into hibernation.

**Pull request closed without merging**